hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -128,15 +127,11 @@ class WorkflowParams(param.Parameterized):
     This class contains all parameters that affect how the whole training and testing workflow is executed.
     """
     random_seed: int = param.Integer(42, doc="The seed to use for all random number generators.")
-    weights_url: List[str] = param.List(default=[], class_=str,
-                                        doc="If provided, a set of urls from which checkpoints will be downloaded"
-                                            "and used for inference.")
-    local_weights_path: List[Path] = param.List(default=[], class_=Path,
-                                                doc="A list of checkpoints paths to use for inference, "
-                                                    "when the job is running outside Azure.")
-    model_id: str = param.String(default="",
```

Comment:
Why do we get rid of model_id in this PR?

Reply:
because it was not used.